spec/models/page_spec.rb

```diff
@@ -11,35 +11,35 @@
 
   describe '#validate_blank_page' do
     let(:page) { build_stubbed(:page) }
-    it 'sets Page::STATUS_BLANK' do
-        page.status = Page::STATUS_BLANK
+    it 'sets :blank' do
+        page.status = :blank
         page.validate_blank_page
 
-        expect(page.status).to eq(Page::STATUS_BLANK)
+        expect(page.status).to eq('blank')
```

Comment:
Here's another case in which I wonder if we can get access to the enums outside of the model, so that we won't have to hard-code string literals.

Reply:
Yes, we can do `Page.statuses[:blank]`. Might have missed this as I had to refactor a lot